src/main/java/io/openshift/launchpad/ui/osio/BoosterDTO.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Eclipse Public License version 1.0, available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package io.openshift.launchpad.ui.osio;
+
+import io.openshift.booster.catalog.Booster;
+import io.openshift.booster.catalog.Mission;
+import io.openshift.booster.catalog.Runtime;
+
+/**
+ */
+public class BoosterDTO
```

Comment:
to minimise the data returned to the client & avoid the rendering issue that seemed to be caused by using Booster directly